python/write_file.py

```diff
@@ -0,0 +1,27 @@
+"""Script to write text to a new file.
+Run script as `python write_file.py <output_file>.py "some text"`
+"""
+import sys
+import os
+
+if len(sys.argv) != 3:
+    print("Don't forget to enter the output file path and a string")
+
+outfile = sys.argv[1]
+text = sys.argv[2]
+
+if os.path.exists(outfile):
+   print("File already exists, clean this up to be sure we're writing/reading a new file")
+
+with open(outfile, 'w') as f:
+    f.write(text.upper())
+
+# Open the file that was just written and print what you see
+with open(outfile, 'r') as f:
+    print(f.readline())
+
+# Clean up the file
+if os.path.exists(outfile):
```

Comment:
We don't need to clean up the file b/c when the container exits, it will all go away.
BUT, if you run the script it locally, it'd be nice for it to clean up after itself, but the test should handle that

Reply:
I also can't figure out how to have the test delete the output file. I get that we don't need to clean it up if running in docker since the container will exit, so then i guess the test doesn't need to do anything. It's just when running it manually that we get the output.txt file. So I guess we just have to manually delete it.

Reply:
> It's just when running it manually that we get the output.txt file.
Maybe we always attempt to delete a local `output.txt` but only if it exists (or try to delete and swallow the error when it doesn't exist). So, if it was run locally, then there would be something to delete, but if run in Docker, then it would "pointlessly" try to delete the file.

Reply:
Also, the deleting of `output.txt` could go at the end of the test, but it'd probably even be better if it went in the beginning of the test to ensure that there is no `output.txt` file leftover from a previous run.

Additionally/alternatively, we could name the file differently every time (like by putting a uuid in the filename). That way, it could never get confused by the output file from a previous run.

Additionally, we could put the output file in a directory (e.g. `f"tmp/output-{uuid}.txt"`) and then if we fail to clean up the files for some reason, it's no big deal-- we can just delete all the files in that directory if they accumulate.

Reply:
In our converasation yesterday we decided to punt on cleaning up the file or naming it w/ a UUID until it's a problem.